frontend/routes/viewerGui/components/previewItemInfo/previewItemInfo.component.tsx

```diff
@@ -59,8 +64,30 @@ export class PreviewItemInfo extends React.PureComponent<IProps, any> {
 		return <>{this.props.actionButton}</>;
 	});
 
+	public renderViewModel = renderWhenTrue(() => {
+		const { type, id } = this.props;
+		const { teamspace, modelId } = this.props.urlParams;
+		return (
+			<OpenInViewerButton
+				preview
+				teamspace={teamspace}
+				model={modelId}
+				query={`${type}Id=${id}`}
+			/>
+		);
+	});
```

Comment:
I'm not sure it was necessary to move the button inside previewItemInfo because the button is using absolute position.

Reply:
fair point !